Source/Applications/SystemCenterNotification/Controllers/OpenXDAControllers.cs

```diff
@@ -425,12 +416,12 @@ public class ScheduledEmailDataSourceController : ModelController<ScheduledEmail
     public class ScheduledEmailDataSourceSettingController : ModelController<ScheduledEmailDataSourceSetting> { }
 
     [RoutePrefix("api/openXDA/TriggeredEmailDataSourceEmailType")]
-    public class TriggeredEmailDataSourceEmaulTypeController : ModelController<TriggeredEmailDataSourceEmailTypeView>
+    public class TriggeredEmailDataSourceEmailTypeController : ModelController<TriggeredEmailDataSourceEmailTypeView>
```

Comment:
```suggestion
    public class TriggeredEmailDataSourceEmailTypeController : ModelController<TriggeredEmailDataSourceEmailTypeView, TriggeredEmailDataSourceEmailType>
```

I would do this then delete the overriden delete method. By providing the second type, it tells the controller to use that for editing, which should fix the problem. Not sure if we can do the same with patch/post, but maybe those too so we can use generic methods.